inc/modules/export/prince/class-printpdf.php

```diff
@@ -85,9 +85,9 @@ protected function themeOptionsOverrides() {
 		$sass = \Pressbooks\Container::get( 'Sass' );
 
 		if ( $sass->isCurrentThemeCompatible( 2 ) ) {
-			$extra = "/* Print Overrides */\n\$prince-image-resolution: 300dpi; \n";
+			$extra = "/* Print Overrides */\n\$prince-image-resolution: auto,normal \n";
```

Comment:
Now that I consider further, we can remove the `themeOptionsOverrides()` method entirely as it should inherit the parent method: https://github.com/pressbooks/pressbooks/blob/05ac44820f1e582dafa89af9ab60521437252e30/inc/modules/export/prince/class-printpdf.php#L80-L119